### PR TITLE
expose log.level for promlog for remote_storage_adapter

### PR DESCRIPTION
**Pull request by sipian**
#3716 
This PR exposes log.level to remote_storage_adapter promlog by taking log.level as a command line argument.
Default log.level is **debug**

### CODE REVIEW

**Comment by silentpete** [comment]
@brian-brazil @gouthamve ,
If you're going to close #4043, should we also close this ticket, #3880, since the hard coded "debug" is in #3635 as well.... the community is asking that this be resolved in some way shape or form and giving you the code to fix it.
Each commit has good things in it... maybe all three should be merged? but at least can we get the ability to choose?

File: documentation/examples/remote_storage/remote_storage_adapter/main.go [hunk 1]
```diff
@@ -141,6 +146,9 @@ func parseFlags() *config {
 	)
 	flag.StringVar(&cfg.listenAddr, "web.listen-address", ":9201", "Address to listen on for web endpoints.")
 	flag.StringVar(&cfg.telemetryPath, "web.telemetry-path", "/metrics", "Address to listen on for web endpoints.")
+	flag.StringVar(&cfg.logLevel, "log.level", "",
+		"The log level to be set for promlog",
```
Comment (silentpete) [inline documentation/examples/remote_storage/remote_storage_adapter/main.go]: @sipian can you add the log levels to this comment, that way when a user uses option '-h' they will know what is available options for this flag?
error, warn, info and debug

**Comment by sipian** [comment]
Is anything more to do with this PR? If not, I think this can be merged now

**Comment by deejay1** [comment]
@sipian you probably need to signoff the commits - see https://github.com/probot/dco#how-it-works

**Comment by silentpete** [comment]
@sipian, Please DCO

**Comment by sipian** [comment]
I seem to have fried up this PR trying to fix DCO. I'll make a new PR with DCO
Sorry !!

**Pull request closed without merging**